internal/httpapi/config.go

```diff
@@ -0,0 +1,15 @@
+package httpapi
+
+type Config struct {
```

Comment:
Consider moving the loading of the configuration to this file (to encapsulate the implementation). Also saving it to a global variable will allow fetching it from everywhere without having to keep readying the env